test/exe/Main.hs

```diff
@@ -2107,14 +2106,14 @@ findDefinitionAndHoverTests = let
   tst (get, check) pos targetRange title = testSessionWithExtraFiles "hover" title $ \dir -> do
 
     -- Dirty the cache to check that definitions work even in the presence of iface files
-    liftIO $ runInDir dir $ do
```

Comment:
The point of this was to ensure that we have a different ghcide session, so that the shake graph is created anew using iface files.

Reply:
I thought it was a leftover. Reverted